src/openms/include/OpenMS/TRANSFORMATIONS/FEATUREFINDER/MultiplexDeltaMassesGenerator.h

```diff
@@ -74,19 +77,55 @@ namespace OpenMS
     void generateKnockoutDeltaMasses();
 
     /**
-     * @brief write the list of labels for each sample
+     * @brief write the list of labels for each of the sample
      */
-    void printLabelsList() const;
+    void printSamplesLabelsList() const;
     
     /**
      * @brief write the list of all mass patterns
      */
-    void printMassPatternList() const;
+    void printDeltaMassesList() const;
+    
+    /**
+     * @brief returns the list of mass shift patterns
+     */
+    std::vector<MultiplexDeltaMasses> getDeltaMassesList();
     
     /**
      * @brief returns the list of mass shift patterns
      */
-    std::vector<MultiplexDeltaMasses> getMassPatternList() const;
+    const std::vector<MultiplexDeltaMasses>& getDeltaMassesList() const;
+    
+    /**
+     * @brief returns the list of samples with their corresponding labels
+     */
+    std::vector<std::vector<String> > getSamplesLabelsList();
+    
+    /**
+     * @brief returns the list of samples with their corresponding labels
+     */
+    const std::vector<std::vector<String> >& getSamplesLabelsList() const;
+    
+    /**
+     * @brief returns the short label string
+     * 
+     * @param label    long label, e.g. "Label:13C(6)15N(4)"
+     */
+    String getLabelShort(String label);
+    
+    /**
+     * @brief returns the long label string
+     * 
+     * @param label    short label, e.g. "Arg10"
+     */
+    String getLabelLong(String label);
```

Comment:
I think choosing long / short is not very descriptive and can be improved. Is it the unimod name?